service/controller/app/resource/chart/current.go

```diff
@@ -27,14 +27,21 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 		return nil, microerror.Mask(err)
 	}
 
-	if cc.Status.TenantCluster.IsDeleting {
+	if cc.Status.ChartStatus.Status != "" {
```

Comment:
How do we know that is the problem? What if we add another type of status message?

Reply:
In most cases, we would add status only for failure, but checking status would do no harm. I agree.